project/mission-vision.md

```diff
@@ -0,0 +1,14 @@
+## Mission
+
+To provide institutions a foundation to support the scientific data
+life-cycle.
+
+## Vision
+
+We will become a focal point for institutions to collaborate, share
+scientific data, integrate new technologies and develop their
```

Comment:
comma after technologies

Reply:
So are you advocating for an Oxford Comma? https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

Reply:
But Weird Al told me not to (https://www.youtube.com/watch?v=8Gv0H-vPoDc)

Reply:
Oh yes! I loooooove the Oxford comma! :)

Reply:
Okay, good enough for me ;)